recipe/patches/0002-avoid-using-macos-12-resource.patch

```diff
@@ -0,0 +1,29 @@
+From 4f04e865ab0e370890a3b2c3c3d5254607abd3a9 Mon Sep 17 00:00:00 2001
+From: Rafael Martins <rmartins@anaconda.com>
+Date: Fri, 28 Jun 2024 23:18:43 +0200
+Subject: [PATCH 2/2] avoid using macos 12 resource
```

Comment:
some explanation as to _why_ it's needed would be helpful for future reference

Reply:
we build with sdk from macos 11. I think that this is easy to grasp from the message, no?

Reply:
oh yeah, I didn't remember that.